### DISCUSSION TITLE
Get values of  multiple set-cookie  response headers

### DISCUSSION THREAD

**Discussion by npackard0** [Budibase/budibase#8543] [Help] [answered]
How do I  get the value of a set-cookie response header where the response returns multiple set-cookie headers? I only see the first set-cooking in the response when I try to create a dynamic variable. attached screen shot of headers in postman vs budibase.
![postman](https://user-images.githubusercontent.com/10286321/200181987-595d3a2d-3b67-4758-abf4-4643f7567624.png)
![budibase](https://user-images.githubusercontent.com/10286321/200181992-49efbcd0-ac3d-4c9c-bf8a-e5bbfab177bb.PNG)

**Comment by melohagan** [comment, marked as answer]
Hey @npackard0 

This is currently not possible as the KeyValue element in Budibase expects the keys to be unique at the moment. 
There is an open related issue here: https://github.com/Budibase/budibase/issues/4844

**Reply by melohagan** [reply]
I also found this comment on StackOverflow which you may be able to make use of, although Budibase probably will escape the characters:

> I'd never suggest this as an answer, but you could try exploit the fact that requests doesn't automaticalle escape the content of the passed headers dict, and pass headers={'X-Attribute': 'name=value\r\nX-Attribute: name=value'}. It's not a very good solution however.
